build/dependencies.props

```diff
@@ -67,6 +67,9 @@
     <DevDependency_MicrosoftExtensionsDependencyModelPackageVersion>2.0.0</DevDependency_MicrosoftExtensionsDependencyModelPackageVersion>
     <DevDependency_WindowsAzureStoragePackageVersion>8.7.0</DevDependency_WindowsAzureStoragePackageVersion>
     <FSharpCorePackageVersion>4.2.1</FSharpCorePackageVersion>
+    <IdentityServer4PackageVersion>2.3.0-preview1-update2</IdentityServer4PackageVersion>
```

Comment:
Has this new dependency been cleared by @Eilon ?

Reply:
Yes